GetIntoTeachingApi/Startup.cs

```diff
@@ -230,5 +236,30 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment hostEnv)
                 endpoints.MapControllers();
             });
         }
+
+        private void ConfigureRateLimiting(IServiceCollection services)
+        {
+            // Load appsettings.json.
+            services.AddOptions();
+
+            // Stores counters/IP rules.
+            services.AddMemoryCache();

```

Comment:
Is there an option to use a Redis cache - in theory load balancing and not much other load you'll end up applying N * limits you've set - where N is the number of nodes

Reply:
Or memcache?

Reply:
Scratch that - just read the README, - RTFM, lol